exporter/loadbalancingexporter/factory.go

```diff
@@ -31,7 +31,9 @@ const (
 
 // NewFactory creates a factory for the exporter.
 func NewFactory() component.ExporterFactory {
-	view.Register(MetricViews()...)
+	if err := view.Register(MetricViews()...); err != nil {
+		log.Fatalln("unable to register given views")
```

Comment:
```suggestion
		log.Fatalf("unable to register given views: %v", err)
```

Also, I wonder if we should change the signature so NewFactory() would return error to make it more elegant. WDYT @dmitryax

Reply:
Too late :)) we have the builder using the NewFactory, and in lots of other places. Also this pattern soon will change when switching to otel/metric